plugins/renamerOnUpdate/README.md

```diff
@@ -1,19 +1,19 @@
 
 # SQLITE Renamer for Stash
-Using metadata from your stash to rename your file.
+Using metadata from your Stash to rename your files.
```

Comment:
The OnUpdate plugin works on a single scene when its updated so `file` should be more accurate